.github/workflows/ci.yml

```diff
@@ -57,7 +57,7 @@ jobs:
       - name: Install Foundry
         uses: foundry-rs/foundry-toolchain@v1
         with:
-          version: nightly-e15e33a07c0920189fc336391f538c3dad53da73
+          version: nightly-d369d2486f85576eec4ca41d277391dfdae21ba7
```

Comment:
I updated foundry version to [this nightly version](https://github.com/foundry-rs/foundry/releases/tag/nightly-d369d2486f85576eec4ca41d277391dfdae21ba7), which includes [this commit version](https://github.com/foundry-rs/foundry/commit/3b1129b5bc43ba22a9bcf4e4323c5a9df0023140) that Optimism is using.